index.html

```diff
@@ -0,0 +1,199 @@
+                  alt="icon2"
+                  class="speaker"
+                />
+              </figure>
+              <div>
+                <h3>Exhibition</h3>
+              </div>
+              <p>
+                Examining the sociology of special education, Tomlinson (1982),
+              </p>
+            </article>
+            <article class="article">
+              <figure>
+                <img
+                  src="./assets/workshop-icon.png"
+                  alt="icon3"
+                  class="speaker"
+                />
+              </figure>
+              <div>
+                <h3>Workshop</h3>
+              </div>
+              <p>
+                “It is important to recognize the distinction between impairment
+              </p>
+            </article>
+            <article class="article">
+              <figure>
+                <img
+                  src="./assets/message-icon.png"
+                  alt="icon4"
+                  class="speaker"
+                />
+              </figure>
+              <div>
+                <h3>Lecture</h3>
+              </div>
+              <p>
+                “It is important to recognize the distinction between impairment
+              </p>
+            </article>
+            <article class="article">
+              <figure>
+                <img
+                  src="./assets/network-icon.png"
+                  alt="icon5"
+                  class="speaker"
+                />
+              </figure>
+              <div>
+                <h3>Network</h3>
+              </div>
+              <p>This included challenging particular forms of psychological</p>
+            </article>
+          </div>
+          <h2>SEE THE WHOLE PROGRAM</h2>
+        </section>
+        <section
+          class="featured-speakers-container"
+          id="featured-speakers"
+        ></section>
+        <section class="partners-list">
+          <h1 class="partn-headg">Partner</h1>
+          <ul class="partners">
+            <li class="partner Mozila">mozila</li>
+            <li class="partner Google">Google</li>
+            <li class="partner Naver">NAVER</li>
+            <li class="partner daumkakao">daumkakao</li>
+            <li class="partner Airtel">Airtel Congo</li>
+          </ul>
+        </section>
+      </main>
+      <footer>
+        <div class="footer-logo">
+          <span
+            ><img
+              src="./assets/cc-icon.png"
+              alt="footer-icon"
+              class="footer-icon"
+          /></span>
+          <h2>
+            creative <br />
+            common corea
+          </h2>
+        </div>
+        <div class="footer-description">
+          <h5>2022 Creative common corea. Some rights Reserved.</h5>
+          <p>Recognize the distinction between impairment</p>
+        </div>
+      </footer>
+    </div>
+
+    <script src="./app.js"></script>
+    <script src="./featured_speaker.js"></script>
```

Comment:
- [ ] The section "Featured speakers" is generated dynamically with JavaScript. I couldn't achieve this. It will be nice to fix this. Kindly check the path.

Reply:
Hey @enaburekhan, thank you for helping me improve my code.

Reply:
@YannickZahinda, Sorry about this highlighted feature. It works as expected. You can go ahead and submit.

Reply:
Thank you @enaburekhan !